res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1272,14 +1281,25 @@ PioneerDDJSX.changeParameters = function(group, ctrl, increment) {
         }
         PioneerDDJSX.selectedSlicerQuantization[deck] = PioneerDDJSX.slicerQuantizations[PioneerDDJSX.selectedSlicerQuantizeParam[deck]];
     }
+    //Slicer Mode + SHIFT:
+    if (ctrl === PioneerDDJSX.nonPadLeds.shiftParameterLeftSlicerMode || ctrl === PioneerDDJSX.nonPadLeds.shiftParameterRightSlicerMode) {
+        // change parameter set:
+        if (ctrl === PioneerDDJSX.nonPadLeds.shiftParameterLeftSlicerMode && PioneerDDJSX.selectedSlicerDomainParam[deck] > 0) {
+            PioneerDDJSX.selectedSlicerDomainParam[deck] = PioneerDDJSX.selectedSlicerDomainParam[deck] - 1;
```

Comment:
You could simplify this a little:
`PioneerDDJSX.selectedSlicerDomainParam[deck] -= 1`